web/api/v1/api_test.go

```diff
@@ -3796,7 +3797,17 @@ type fakeDB struct {
 	err error
 }
 
-func (f *fakeDB) CleanTombstones() error                                         { return f.err }
+func (f *fakeDB) CleanTombstones() error { return f.err }
+
+func (f *fakeDB) ListBlockMetas() ([]tsdb.BlockMeta, error) {
```

Comment:
Let's also add a test to `api_test.go` for the new endpoint I'd say. Should hopefully be pretty simple and not need multiple cases.